server/services/whatsNewCookieService.ts

```diff
@@ -1,12 +1,15 @@
 import { Request, Response } from 'express'
+import md5 from 'md5'
 
 export default class WhatsNewCookieService {
   static COOKIE_NAME = 'whats-new-banner'
 
-  static persistDismissedVersion = (res: Response, version: number) => {
+  static key = (id: string) => `${this.COOKIE_NAME}-${md5(id)}`
```

Comment:
You could have hashed the whole string like `md5(${this.COOKIE_NAME}-${id})`

Reply:
Done @kth13